data_validation/validation_builder.py

```diff
@@ -358,3 +389,38 @@ def add_query_limit(self):
         limit = self.config_manager.query_limit
         self.source_builder.limit = limit
         self.target_builder.limit = limit
+
+    def get_query_from_file(self,filename):
+        """ Return query from input file """
+        query = ""
+        try:
+            file = open(filename, "r")
+            query = file.read()
+        except IOError:
+            print("Cannot read query file: ",filename)
+
+        if not query or query.isspace():
+            raise ValueError(
+            "Expected file with sql query, got empty file or file with white spaces. "
+            f"input file: {filename}"
+            )
+        file.close()
+        return query
+
+    def get_aggregation_query(self, agg_type, column_name):
```

Comment:
@renzokuken Curious on your thoughts on this and if there's a better way to generate this SQL as a string. This is so that the user can run a sum/count/min/max on the source & target query results.